axes/attempts.py

```diff
@@ -125,7 +129,7 @@ def get_user_attempts(request):
     # If objects were deleted, we need to update the queryset to reflect this,
     # so force a reload.
     if force_reload:
-        attempts = _query_user_attempts(request)
+        attempts = _query_user_attempts(request, username)
```

Comment:
should move away from using request.method period.

should update the 
```
        settings.AXES_ONLY_USER_FAILURES or
        settings.AXES_LOCK_OUT_BY_COMBINATION_USER_AND_IP
    ) and request.method == 'GET':
        return False
```
id is_already_login to not check the get and not check the post anywhere. 

I can't conceive of any way to call this that wouldn't have credentials available. Its reasonable to expect the caller to specify a username.